tests/src/JIT/CodeGenBringUpTests/Rotate.cs

```diff
@@ -128,12 +146,30 @@ static ulong ror64comm(ulong value, int amount)
     [MethodImpl(MethodImplOptions.NoInlining)]
     static ulong ror64const()
     {
-        ulong value = flag() ? (ulong)0x123456789abcdef : (ulong)0x123456789abcdef;
+        ulong value = flag() ? (ulong)0x123456789abcdef : (ulong)0xabcdef123456789;
         int amount = flag() ? 5 : 5;
```

Comment:
I'm not sure. @erozenfeld wrote the initial test. May just be a copy-paste error.